packages/config/maintenance/importConfig.ts

```diff
@@ -1388,7 +1388,7 @@ async function parseZWAProduct(
 				}
 			}
 		}
-		newConfig.paramInformation[param.ParameterNumber] = parsedParam;
+		if (!found) newConfig.paramInformation.push(parsedParam);
```

Comment:
Is this right? Doesn't this make it so that we only push the parameter if there was not already a matching param in the original file, which means we can never update what is there? We've been purposely merging the zwa parameters over what we have otherwise there is no way to see errors in our current files. It is also easier to matchup for templating if you abandon our current description/label because the manufacturers tend to use similar labels and such for Levenshtein distance calcs.

Also, since newConfig is what we're writing back, wouldn't we lose all parameters that presently exist?

I think we should always push the parameter if I understand what this is doing.